packages/shared/src/components/squads/PublicSquadEligibilityCard.tsx

```diff
@@ -10,42 +16,46 @@ import { useLazyModal } from '../../hooks/useLazyModal';
 import { LazyModal } from '../modals/common/types';
 import { anchorDefaultRel } from '../../lib/strings';
 import {
-  SquadPostsProgressBar,
   MIN_SQUAD_POSTS,
+  SquadPostsProgressBar,
 } from './SquadPostsProgressBar';
 import { squadsPublicGuide } from '../../lib/constants';
+import { SquadStatus } from './settings';
 
-interface Props {
-  postsCount: number;
-  inReview: boolean;
-  squadId: string;
+interface PublicSquadEligibilityCardProps {
+  placeholder: ReactNode;
```

Comment:
Not sure this should be passible?
Also now that I think about it, you tested support for new list view as well?